File: dsa/2LinearSearch/SearchInRange.java
public class SearchInRange {
    public static void main(String[] args) {
        int[] nums = {9,20,35,73,46,2,99};
    }
    static boolean search(int[] arr,int target,int start,int end){

        if (arr.length == 0) {
            return false;
        }

        for(int i =start;i<=end;i++ ){
            if (arr[i] == target) {
                return true;
            }
        }

        return false;
    }
}

File: dsaRecent/LinearSearch/SearchInRange.py
def SearchInRange(arr,start,end,target):
    for i in range(start,end+1):
        if arr[i] == target:
            return i
    return -1

arr = [9,20,35,73,1,43,5,99]
isPresentInRange = SearchInRange(arr,2, 5,73)
if isPresentInRange != -1:
    print(f"present at inde {isPresentInRange}")
else:
    print("not present")